python/kvikio/tests/test_benchmarks.py

```diff
@@ -78,3 +80,60 @@ def test_zarr_io(run_cmd, tmp_path, api):
         cwd=benchmarks_path,
     )
     assert retcode == 0
+
+
+def skipif_libcudf_s3_io_option_is_not_available() -> None:
+    """Call pytest.skip() if cudf or its "libcudf_s3_io" option isn't available
+
+    See <https://github.com/rapidsai/cudf/pull/16499>
+    """
+    cudf = pytest.importorskip("cudf")
+    try:
+        cudf.get_option("libcudf_s3_io")
+    except KeyError:
+        pytest.skip(
+            """cudf doesn't has the "libcudf_s3_io" option, """
```

Comment:
Suggesting a simpler quoting style here.
```suggestion
            "cudf doesn't have the 'libcudf_s3_io' option, "
```